x/consortium/spec/01_concepts.md

```diff
@@ -0,0 +1,56 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Authorization
+
+The consortium module is designed to contain the authorization information. The other modules may deny its message based on the information of the consortium. As of now, the following modules are using the information:
+
+- **[Staking Plus](../../stakingplus/spec/README.md)**
+    - [Msg/CreateValidator](../../stakingplus/spec/03_messages.md#msgcreatevalidator)
+
+One can update the authorization, via proposals:
+
+- `UpdateValidatorAuthsProposal` to authorize `Msg/CreateValidator`
+
+    +++ https://github.com/line/lbm-sdk/blob/v0.44.0-rc0/proto/lbm/consortium/v1/consortium.proto#L31-L40
```

Comment:
How about using latest version of lbm-sdk?

Reply:
We cannot change the links on every time we release a new version. If we use the latest permanent link, on the other hand, old docs would reference the new code lines, so we cannot simply use the latest link.

So, one plausible option I come up with, is changing the version (and code lines), on each time there is a change in the specification.

Reply:
I meant the latest version is current latest version 0.45.0, not 0.44.0-rc0.

Reply:
The oldest version including x/consortium is 0.44.0-rc0. And there was no change, so I decided to choose that version for the reference.

> So, one plausible option I come up with, is changing the version (and code lines), on each time there is a change in the specification.

If you agree with my suggestion, the version would be always the oldest version since the change.